GoogleForCreators/web-stories-wp
# Dashboard: Update filter bar to use Design System

## Context
Part of updating the dashboard to use the new design system styles

## Summary
Updates this bar in the dashboard navigation:
(note: these are all the same width, I just didn't take screen shots with consistent dimensions)
**old**
<img width="982" alt="Screen Shot 2021-02-17 at 2 57 36 PM" src="https://user-images.githubusercontent.com/35983235/108273233-82195800-7130-11eb-8eb8-372c20e22592.png">
<img width="1009" alt="Screen Shot 2021-02-17 at 2 57 29 PM" src="https://user-images.githubusercontent.com/35983235/108273238-82195800-7130-11eb-972d-dac9e8c49ca7.png">

**new**
<img width="1102" alt="Screen Shot 2021-02-17 at 2 48 18 PM" src="https://user-images.githubusercontent.com/35983235/108273018-48e0e800-7130-11eb-9d6f-09d186c1173a.png">
<img width="1053" alt="Screen Shot 2021-02-17 at 2 48 04 PM" src="https://user-images.githubusercontent.com/35983235/108273020-49797e80-7130-11eb-8832-aebed220bac5.png">

Designs: https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?version-id=683762501&node-id=25%3A1312

## Relevant Technical Choices
NA


## To-do
NA


## User-facing changes
Can be seen above in summary.


## Testing Instructions
See that you can still reorder your stories as specified and toggle between grid and table views


### QA






This PR can be tested by following these steps:

1. Follow testing instructions above

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #4963
